packages/start/src/server/spa/StartServer.tsx

```diff
@@ -18,12 +18,14 @@ export function StartServer(props: { document: Component<DocumentComponentProps>
   const context = getRequestEvent() as PageEvent;
   // @ts-ignore
   const nonce = context.nonce;
+  useAssets(context.assets, nonce);
+
   return (
     <NoHydration>
       {docType as unknown as any}
       <TopErrorBoundary>
         <props.document
-          assets={<>{context.assets.map((m: any) => renderAsset(m))}</>}
+          assets={<></>}
```

Comment:
is this supposed to empty? I guess the useAssets does it

Reply:
Yuppa :), Solids/dom-expressions `useAssets` injects the assets directly in renderToString at the end of the head (<https://github.com/ryansolid/dom-expressions/blob/2a3de6623e10da44adceb3aaa718cf4325296576/packages/dom-expressions/src/server.js#L577C10-L577C22>).

I guess we could remove the "assets" prop here, but I am a bit hesitant if its 100% the right move 🤔